app/src/main/res/values/strings.xml

```diff
@@ -35,6 +37,7 @@
     <string name="errorMsg">Sorry, something went wrong!</string>
     <string name="choose_photo">Choose photo</string>
     <string name="upload_voucher_image">Upload Voucher Image</string>
+    <string name="info">ingo</string>
```

Comment:
misspelling. also what is this string used for? it seems a little weird if it's something you're showing to the user

Reply:
this layout is for the top part of the swipe up view on the maps page
not sure I understand your question; do you mean that it's strange to have a text view with just "info" in it? not super sure what the point of it was, since Mia was working on this layout.

Reply:
> do you mean that it's strange to have a text view with just "info" in it?

yeah, that's what i meant. ok, no worries! but it should definitely be info not ingo :P